### PR TITLE
task3/html - user information validation

### PR DESCRIPTION
**Pull request by qnislavnachev**
there is a few text fields that should have javascript validations
if some of information doesnt match it returns alert message

### CODE REVIEW

**Comment by qnislavnachev** [comment]
here is a demo of this task: https://qnislavnachev.github.io/logPage.html

**Comment by tsony-tsonev** [comment]
You should display the errors in a span (red colored text) next to the input or below it. Also you try using regex for the validation so you can remove the if/else logic. This will allow you to reuse same validation logic for almost all of the inputs.

**Comment by qnislavnachev** [comment]
last fixes are ready for review using regex for validation and display errors in span

File: task3/logPageScript.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+function validation(regex, value, errorId, message) {
+	if (!regex.test(value)) {
+		document.getElementById(errorId).innerHTML = message;
+		return;
+	}
+	document.getElementById(errorId).innerHTML = "";
+}
+
+function validateConfirmPass(form) {
```
Comment (tsony-tsonev) [inline task3/logPageScript.js]: why not pass the two passwords instead of the whole form

File: task3/logPageScript.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	var pass = form.pass.value;
+	var confirmPass = form.confirmpass.value;
+	if (pass == confirmPass) {
+		document.getElementById("confirmpassError").innerHTML = "";
+	} else {
+		document.getElementById("confirmpassError").innerHTML = "Password does not match";
+	}
```
Comment (tsony-tsonev) [inline task3/logPageScript.js]: here you can try to remove the else with the approach from the method above

**Comment by tsony-tsonev** [comment]
LGTM when you fix the comments

**Pull request closed without merging**